store/kv/kv_store.go

```diff
@@ -79,26 +78,37 @@ func (tx *KeyValueTx) Read(ctx context.Context, table []byte, key Key) (Iterator
 	if err != nil {
 		return nil, err
 	}
-	return &KeyValueIterator{
-		baseIterator: iter,
-	}, nil
+	// TODO: move this to TxImplWithMetrics to kv/measure.go
+	if config.DefaultConfig.Metrics.Fdb.Enabled || config.DefaultConfig.GlobalStatus.Enabled {
+		return NewKeyValueIteratorWithMetrics(ctx, iter), nil
```

Comment:
If we move this to measure then this `if` check is only needed in main.go during the initialization kv.

Reply:
This is a bit more complicated than I thought, we can sync about it and push this change separately.